src/Core/Console.php

```diff
@@ -14,6 +14,7 @@ class Console extends \Asika\SimpleConsole\Console
 	protected $customHelpOptions = ['h', 'help', '?'];
 
 	protected $subConsoles = [
+		'blockedservers'         => __NAMESPACE__ . '\Console\BlockedServers',
```

Comment:
The other block command is called `globalcommunityblock`, maybe we should name this `serverblock`?